src/clincoded/static/components/variant_central/interpretation/shared/extra_evidence.js

```diff
@@ -285,17 +285,17 @@ var ExtraEvidenceTable = module.exports.ExtraEvidenceTable = createReactClass({
             // deep copy the evidence data
             data = JSON.parse(JSON.stringify(foundCandidate));
             data.source.data['comments'] = (foundCandidate.source.data['comments'] && foundCandidate.source.data['comments'] != '') ?
```

Comment:
Should these references to `source` be `sourceInfo` as well?

Reply:
Updated the references.  Thanks.